index.js

```diff
@@ -249,4 +252,26 @@ class Analytics {
   }
 }
 
+var retryCondition = function (error) {
```

Comment:
Should be an arrow function + `const` instead of `var`.

Reply:
I also might suggest adding this method to the Analytics class so we can more easily write tests for it.

Reply:
what's an arrow function?

Reply:
Updated to be `const`.

Reply:
> I also might suggest adding this method to the Analytics class so we can more easily write tests for it.

Why would that be easier? This seems like a stateless function that can be tested without depending on the analytics class.

Also per my understanding, adding it to the analytics class will automatically expose this function to the public API.

Reply:
In order to test it directly, we'd have to export it. Attaching it directly to `exports` (or `module.exports`) make it _look_ like part of the public API. If we add it as a method of the class, it will be exposed, but won't _look_ like it's part of the public API.

Consider the following:

```js
// exporting directly
exports.somePrivateMethod = () => {}

// exporting as part of the class
module.exports = class Analytics {
  track () {}
  somePrivateMethod () {}
}
```

IMO the first _looks_ like it's exposed for consumption by the end user, while the second just looks like a regular class.

Reply:
> what's an arrow function?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
Thanks! Makes sense about testing, chatted with Roland and he suggested using _ as a prefix to denote it as a private, so I'll do that.